src/proposals/20250123/SparkEthereum_20250123.sol

```diff
@@ -0,0 +1,121 @@
+    address constant public SPARKLEND_USDC_ATOKEN  = 0x377C3bd93f2a2984E1E7bE6A5C22c525eD4A4815;
+    // Same oracle composed with chi oracle on 2024-10-17.
+    address constant public FIXED_1USD_ORACLE      = 0x42a03F81dd8A1cEcD746dc262e4d1CD9fD39F777;
+    address constant public USDS_IRM               = 0x2DB2f1eE78b4e0ad5AaF44969E2E8f563437f34C;
+
+    function newListings() public pure override returns (IEngine.Listing[] memory) {
+        IEngine.Listing[] memory listings = new IEngine.Listing[](1);
+        listings[0] = IEngine.Listing({
+            asset:       Ethereum.USDS,
+            assetSymbol: 'USDS',
+            priceFeed:   FIXED_1USD_ORACLE,
+            // Deploying the default one the listing engine uses out of
+            // convenience, will overwrite it in  _postExecute
+            rateStrategyParams:                Rates.RateStrategyParams({
+                optimalUsageRatio:             0,
+                baseVariableBorrowRate:        0,
+                variableRateSlope1:            0,
+                variableRateSlope2:            0,
+                stableRateSlope1:              0,
+                stableRateSlope2:              0,
+                baseStableRateOffset:          0,
+                stableRateExcessOffset:        0,
+                optimalStableToTotalDebtRatio: 0
+            }), 
+            enabledToBorrow:       EngineFlags.ENABLED,
+            stableRateModeEnabled: EngineFlags.DISABLED,
+            borrowableInIsolation: EngineFlags.ENABLED,
+            withSiloedBorrowing:   EngineFlags.DISABLED,
+            flashloanable:         EngineFlags.DISABLED,
+            ltv:                   0,
+            liqThreshold:          0,
+            liqBonus:              0,
+            reserveFactor:         1, // Overriden in _postExecute
+            supplyCap:             0,
+            borrowCap:             0,
+            debtCeiling:           0,
+            liqProtocolFee:        0,
+            eModeCategory:         0
+        });
+        return listings;
+    }
+
+    function _postExecute() internal override {
+        _onboardAaveToken(
+            SPARKLEND_USDC_ATOKEN,
+            20_000_000e6,
+            uint256(10_000_000e6) / 1 days
+        );
+        _onboardAaveToken(
+            AAVE_PRIME_USDS_ATOKEN,
+            50_000_000e18,
+            uint256(50_000_000e18) / 1 days
+        );
+
+        // Set custom IRM following SSR
+        LISTING_ENGINE.POOL_CONFIGURATOR().setReserveInterestRateStrategyAddress(
+            Ethereum.USDS,
+            USDS_IRM
+        );
+        // Configure USDS market in ways not allowed by the listing engine
+        LISTING_ENGINE.POOL_CONFIGURATOR().setReserveFactor(Ethereum.USDS, 0);
+
+        // Seed the newly listed pool
+        IERC20(Ethereum.USDS).approve(address(LISTING_ENGINE.POOL()), 1e18);
+        LISTING_ENGINE.POOL().supply(Ethereum.USDS, 1e18, address(this), 0);
```

Comment:
Is it instead of `deposit()` function to prevent token value manipulation?

Reply:
`deposit` is deprecated. They both do the same thing.